Makefile.ibi

```diff
@@ -27,29 +27,27 @@ LIBVIRT_POOL = default
 KCLI_IMAGE = quay.io/karmab/kcli
 kcli = podman run --net host -i --rm --security-opt label=disable -v $(LIBVIRT_IMAGE_PATH):$(LIBVIRT_IMAGE_PATH) -v $(HOME)/.ssh:/root/.ssh -v $(HOME)/.kcli:/root/.kcli -v /var/run/libvirt:/var/run/libvirt $(KCLI_IMAGE)
 
-$(RHCOS_LIVE_ISO):
-	curl -L $(RHCOS_LIVE_ISO_URL) > $@
-
-.PHONY: ibi-ignition.json
-ibi-ignition.json: credentials/pull-secret.json credentials/backup-secret.json $(SSH_KEY_PUB_PATH)
-	SSH_PUBLIC_KEY="$(shell cat $(SSH_KEY_PUB_PATH))" \
-	SEED_IMAGE="$(SEED_IMAGE)" \
-	SEED_VERSION="$(SEED_VERSION)" \
-	LCA_IMAGE="$(LCA_IMAGE)" \
-	INSTALLATION_DISK="$(IBI_INSTALLATION_DISK)" \
-	  envsubst < ibi-butane-ignition.yaml | \
-		podman run -i -v $(shell pwd):/data/:rw,Z  --rm quay.io/coreos/butane:release --pretty --strict -d /data > $@
+$(IB_CLI):
+	mkdir -p bin
+	podman run -it -v `pwd`:/data --entrypoint cp $(LCA_IMAGE) /usr/local/bin/ib-cli ./data/bin/
 
 .PHONY: ibi-iso
-ibi-iso: $(RHCOS_LIVE_ISO) ibi-ignition.json ## Embed ignition into live ISO
-	-rm -f rhcos-ibi.iso
-	podman run -v $(shell pwd):/data:rw,Z $(COREOS_INSTALLER) iso ignition embed -i /data/ibi-ignition.json -o /data/rhcos-ibi.iso /data/$(RHCOS_LIVE_ISO)
-	cp rhcos-ibi.iso "$(IBI_RHCOS_ISO_PATH)"
-	chmod a+r "$(IBI_RHCOS_ISO_PATH)"
+ibi-iso: $(IB_CLI) credentials/pull-secret.json credentials/backup-secret.json credentials/ssh_public.key
+	mkdir -p $(IBI_WORK_DIR)
+	$(IB_CLI) create-iso --installation-disk $(IBI_INSTALLATION_DISK) \
```

Comment:
Can we run this inside a container instead of copying lca-cli outside?

Reply:
Whenever I try I find myself adding more and more mounts, this seems simpler